src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectEngineHost/ProjectEngineFactories.cs

```diff
@@ -3,7 +3,7 @@
 
 using System.Collections.Immutable;
 
-namespace Microsoft.AspNetCore.Razor.ProjectEngineHost;
+namespace Microsoft.CodeAnalysis.Razor.Workspaces.ProjectEngineHost;
```

Comment:
This can be fixed later, but we generally don't add "Workspaces" into namespaces in MS.CA.Razor.Workspaces anymore. This should probably have been Microsoft.CodeAnalysis.Razor.ProjectEngineHost, but please feel free to fix that later or tell somebody else to do it. :smile: